src/js/base/IconLink.tsx

```diff
@@ -14,7 +14,7 @@ type IconLinkProps = {
 export function IconLink({ color, name, replace, tip, to, ...props }: IconLinkProps) {
     return (
         <Link replace={replace} to={to} aria-label={props["aria-label"]}>
-            <Icon color={color} name={name} tip={tip} hoverable />
+            <Icon as="button" color={color} name={name} tip={tip} hoverable />
```

Comment:
Why the switch up here? The icon has no inherent functionality here, so I my instinct here is that it is semantically incorrect to render it as a button. If your going for a look it is probably best to take advantage of classname and apply additional styles